Mail2Bug/WorkItemManagement/IWorkItemManager.cs

```diff
@@ -21,5 +21,12 @@ public interface IWorkItemManager
         void ModifyWorkItem(int workItemId, string comment, Dictionary<string, string> values);
 
         INameResolver GetNameResolver();
+
+        /// <summary>
+        /// Get work item for the given ID.
+        /// </summary>
+        /// <param name="workItemId">the work item ID</param>
+        /// <returns>work item corresponding to the ID</returns>
+        SimpleWorkItem GetWorkItem(int workItemId);
```

Comment:
Done. Had to create an interface IWorkItemFields since we are now returning fields on demand and this requires a provider specific implementation. Added TfsWorkItemFields which implements this for TFS/VSO.